flow/Makefile

```diff
@@ -105,7 +105,6 @@ ifneq ($(BLOCKS),)
   export ADDITIONAL_LEFS += $(BLOCK_LEFS)
   export ADDITIONAL_LIBS += $(BLOCK_LIBS)
   export ADDITIONAL_GDS += $(BLOCK_GDS)
-  export GDS_FILES += $(BLOCK_GDS)
   ifneq ($(CDL_FILES),)
```

Comment:
@maliberty Isn't this a standalone bugfix? Split out into separate PR?

I'm not very familiar with the gds part of the flow.

Reply:
Yes, platforms should set `GDS_FILES` and  use `ADDITIONAL_GDS`.